src/content/controller_content.ts

```diff
@@ -153,17 +168,10 @@ function* ParserController() {
 
                 const newMode = contentState.mode
                 if (newMode !== currentMode) {
-                    keyEvents = []
+                    keyEvents = keyEvents.slice(-1)
```

Comment:
Could you explain what you're trying to do in this line?

Reply:
Before it was `check mode -> clear events if it changed-> push new event`, now i grouped KeyboardEvent-related things above including push, and thus instead of clearing we leave the one we pushed